src/app/docs/components/toggle/labeled-toggle/page.mdx

```diff
@@ -0,0 +1,8 @@
+export const metadata = {
+  title: 'Toggle',
+  description: 'A toggle switch where the handle switches between two states.',
+}
```

Comment:
Can you update the title and description to fit your new toggle instead of just Toggle?